jbpm-simulation/src/main/java/org/jbpm/simulation/PathContext.java

```diff
@@ -15,14 +15,15 @@
 
 package org.jbpm.simulation;
 
+
+import org.eclipse.bpmn2.FlowElement;
+import org.eclipse.bpmn2.SequenceFlow;
```

Comment:
Nit-picking, but this is still wrong. This does not follow the `java.*` + `javax.*` first (which is what we should use), nor the alphabetical order (which may be colliding with what we use).